packages/starterkit-handlebars-demo/README.md

```diff
@@ -4,9 +4,9 @@ The Demo StarterKit for Handlebars is meant to be used as a demonstration of a H
 
 ## Requirements
 
-The Base StarterKit for Mustache requires the following PatternEngine:
+The Vanilla StarterKit for Handlebars requires the following PatternEngine:
```

Comment:
Is there a reason you changed Base to Vanilla? I think base is probably a better definition for those not familiar with vanilla being "bland" or "normal"

Reply:
> Is there a reason you changed Base to Vanilla? I think base is probably a better definition for those not familiar with vanilla being "bland" or "normal"

I've adapted it from this file: https://github.com/pattern-lab/starterkit-handlebars-vanilla/blob/master/README.md - but that's incorrect for this part, ob